README.EN.md

```diff
@@ -0,0 +1,65 @@
+
```

Comment:
What was your consideration putting this Readme at the top-level directory? I think it makes sense to move this file to the translations directory as well. See https://github.com/corona-warn-app/cwa-documentation/blob/master/translations/README.de.md

Reply:
Oops we missed that one ;)